src/newgrf_roadstop.cpp

```diff
@@ -171,6 +171,22 @@ uint32 RoadStopScopeResolver::GetVariable(byte variable, uint32 parameter, bool
 			return ssl.grfid;
 		}
 
+		/* 16 bit road stop ID of nearby tiles */
+		case 0x6B: {
+			TileIndex nearby_tile = GetNearbyTile(parameter, this->tile);
+			uint32 grfid = this->st->roadstop_speclist[GetCustomRoadStopSpecIndex(this->tile)].grfid;
```

Comment:
Could/should be moved to right before usage, to save some CPU cycles unless actually needed.